src/libsystemd-network/test-dhcp-client.c

```diff
@@ -56,7 +56,8 @@ static void test_request_basic(sd_event *e) {
         if (verbose)
                 printf("* %s\n", __FUNCTION__);
 
-        r = sd_dhcp_client_new(&client);
+        /* Initialize client without Anonymize settings. */
+        r = sd_dhcp_client_new(&client, 0);
```

Comment:
I'd pass this as "false" here... after all the compiler will make no difference anyway, as bools degrade to int automatically... In public APIs we use C89 "int" for booleans, even though we use C99 "bool" everywhere else, and I figure that means in this case writing "flase" is much nicer to read, even if the parameter is actually an int because this is part of a supposedly eventually to be made public API